src/main/kotlin/app/ui/OpenState.kt

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.ui
+
+import app.Configurator
+
+/**
+ * On application open console UI state.
+ */
+class OpenState constructor(val context: Context) : ConsoleState {
+    override fun doAction() {
+        if (Configurator.config.username.isEmpty()
+                || Configurator.config.password.isEmpty()) {
+            println("Sourcerer hashes your git repositories into intelligent "
+                    + "engineering profiles. If you doesn't have an account, "
```

Comment:
If you don't...

But why do you print the same message from two different places of code?